llvm/lib/Target/AArch64/AsmParser/AArch64AsmParser.cpp

```diff
@@ -5335,8 +5461,8 @@ bool AArch64AsmParser::validateInstruction(MCInst &Inst, SMLoc &IDLoc,
             (Reg >= AArch64::S16 && Reg <= AArch64::S31) ||
             (Reg >= AArch64::H16 && Reg <= AArch64::H31) ||
             (Reg >= AArch64::B16 && Reg <= AArch64::B31)) {
-          Warning(IDLoc, "this instruction uses disallowed registers.");
-          break;
+          Warning(IDLoc, "register " + getArm64ECDisallowedRegName(Reg) +
```

Comment:
```suggestion
          Warning(IDLoc, "register " + RI->getName(Reg) +
```

Reply:
I was looking for something like this, but I couldn't find it - I'm really new to the codebase... thank you!